### PR TITLE
[Improvement] Directly Throw exception when taskInstancy log path is empty which log need to be queried

### PR DESCRIPTION
**Pull request by ruanwenjun**
## Purpose of the pull request

Avoid send RPC to worker, cause worker NPE

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java [hunk 1]
```diff
@@ -131,7 +131,7 @@ public enum Status {
     VERIFY_USERNAME_ERROR(10100, "verify username error", "用户名验证错误"),
     UNAUTHORIZED_USER_ERROR(10101, "unauthorized user error", "查询未授权用户错误"),
     AUTHORIZED_USER_ERROR(10102, "authorized user error", "查询授权用户错误"),
-    QUERY_TASK_INSTANCE_LOG_ERROR(10103, "view task instance log error", "查询任务实例日志错误"),
+    QUERY_TASK_INSTANCE_LOG_ERROR(10103, "view task instance log error: {0}", "查询任务实例日志错误"),
```
Comment (fuchanghai) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java]: hi @ruanwenjun Should we make a slight modification and replace it with ```查询任务实例日志错误: {0}```

Comment (ruanwenjun) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java]: Done

**Review by fuchanghai — DISMISSED** [review]
LGTM

**Review by fuchanghai — APPROVED** [review]
LGTM

**Pull request merged**